spec/initializer/attr_readers_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe Interactor::Initializer::AttrReaders do
+  describe '.for' do
+    subject { described_class.for(dummy_class, attributes) }
+    let(:dummy_class) { Class.new }
```

Comment:
what if `subject` returned the initialized class, then the specs could check `expect(subject).to ...`
Ruby is leaning to returning methods on definition, so returning the class on definition (initialization) would follow the same principle